unitylibs/utils/experiment-provider.js

```diff
@@ -0,0 +1,29 @@
+export default async function getExperimentData() {
+  const decisionScopes = ['ACOM_UNITY_ACROBAT_EDITPDF_POC'];
```

Comment:
I think this was briefly touched on before but wanted to ask again, could this be a value inputted via authoring? Possibly via a query param maybe if not within the block? This is currently very specific to Edit PDF, but want to ensure we have a scalable solution in mind for future tests.

Note: I am not requesting any changes to this current PR as we have a strict deadline for this test. Just wanted to call this out and ensure we have thoughts on how to improve this further after this initial test.